### PR TITLE
Update form bottom button bar appearance 

### PR DESCRIPTION
**Pull request by janfaracik**
This PR is the third of several follow ups to https://github.com/jenkinsci/jenkins/pull/5923, the intention of this PR is to update the appearance of the sticky bottom button bar (commonly seen on longer forms).

Kapture 2022-02-21 at 11 06 33

* The new bar 'floats' above the form now with a soft shadow which tapers at the end
* The new bar now uses built in CSS to do the positioning (using `position: sticky`) - this means less custom JS + better performance
* Tiny table size fix (the table was 10px too small)
* The `quote` class (used to show the home directory path) now uses the monospace font variable for consistency

### Proposed changelog entries

* Update form bottom button bar appearance 

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: war/src/main/less/modules/table.less
Comment (NotMyFault) [inline war/src/main/less/modules/table.less:4]: See right part of the table

**Review by NotMyFault — CHANGES_REQUESTED** [review]

File: war/src/main/less/modules/table.less
Comment (janfaracik) [reply war/src/main/less/modules/table.less:4]: Awesome spot :) Should be fixed now if you wouldn't mind trying again :)

**Comment by timja** [comment]
Needs ATH run, this area is very susceptible to faillures

**Review by fqueiruga — APPROVED** [review]
Looks lovely! The positioning was a bit of a hack. 

@janfaracik there used to be a lot of problems with ATH tests being flaky because the buttons would not update their position. It's reflected here: https://issues.jenkins.io/browse/JENKINS-64177. I would suggest a run of ATH to verify that this is OK

**Comment by timja** [comment]
> I would suggest a run of ATH to verify that this is OK

It's running here: https://github.com/jenkinsci/acceptance-test-harness/pull/746

**Review by NotMyFault — APPROVED** [review]
Works great so far 👍🏻

**Comment by timja** [comment]
ATH passed
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

File: core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.groovy
Comment (daniel-beck) [inline core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.groovy:24]: What happens to forms in plugins that do not have this `class`?

Comment (janfaracik) [reply core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.groovy:24]: The bottom bar fills the width of the page rather than ends at the width of the form, so nothing major just the shadow is wider than it should be.

Comment (daniel-beck) [reply core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.groovy:24]: Why isn't every `f:form` a `jenkins-form`?

**Pull request merged**
